scripts/client/_otter_env_core.sh

```diff
@@ -134,12 +134,11 @@ case "$osname" in
     Darwin)
         anasoft_distro=
         otter_perl=
-        PERL5LIB="${PERL5LIB}:\
-$anasoft/lib/site_perl:\
-$anasoft/lib/perl5/site_perl:\
-$anasoft/lib/perl5/vendor_perl:\
-$anasoft/lib/perl5\
-"
+        #This is based on someone having a Mac will have the app installed. Hopefully it should work even
+        #if it is not installed.
+        #I used '=' for sed but anything can be used except '/'
+        perl_inc=`$anasoft/bin/perl -e '$"=":";print qq(@INC)' | sed "s=/Users[^:]*\.app/Contents/Resources=$anasoft=g"`
```

Comment:
Do we need both perl and sed here? It feels like it should be simple enough to have perl handle regex substitution as well.

Reply:
I would keep perl and sed as it works nicely on one line. By using perl only I would need a loop and to store some variables and I would go in the hell of escaping to get variables at the right time which would make the whole command harder to read.